src/generate/select.ml

```diff
@@ -0,0 +1,35 @@
+let dump_file file =
```

Comment:
If this unmodified from Jane Street's library, I think it needs to retain the copyright header, and have a mention in `ORIGINS.md`, specifying what license it was originally written under.

Reply:
It was slightly modified for better compatibility. There was no copyright header but I has added to ORIGINS.md.

Reply:
Thank you.